### ISSUE TITLE
Enhancement for Repository Language

### ISSUE THREAD

**Issue by FlorianTaut** [gitpoint/git-point#165]
Hey,

i just got an idea how we could show the repo language.

I had the idea we do it just like Github does.

We add a small bar at the end of the grey bar at the top.
This bar contains the colors and just looks similar to the one Github uses:
![unbenannt](https://user-images.githubusercontent.com/18403881/28586814-3c6a0656-7175-11e7-8a29-54e43ac93a73.PNG)

When the user clicks on it we make a small like "dropup" 😆  which shows the following:
![unbenannt1](https://user-images.githubusercontent.com/18403881/28586847-5c4f2082-7175-11e7-9f88-7859ed956972.PNG)

I would love to try it by myself but have no idea how to...

What do you think @housseindjirdeh @andrewda @alejandronanez 

Thank you guys!

**Comment by andrewda** [comment]
That sounds like a really good idea!

**Comment by FlorianTaut** [comment]
When I think over it.. 

A problem could or will be the space. 

What if we have a repository with f.e. 8 different languages. With the colors it will be no problem but the text would be too long for the screen.

**Comment by housseindjirdeh** [comment]
I definitely think this could look nice, but how are you envisioning it exactly? In the repository profile screen - a bar that goes at the very top where tapping it on/off shows all the languages? Just trying to think design wise if a bar that goes across the top part of the screen would look like it fit in the black repository profile component 🤔

**Comment by andrewda** [comment]
I was actually thinking that across the bottom might look better. Just a very thin line with the repository language colors. Maybe we could have a new screen open up when the line is clicked with a ScrollView of the different repository languages?

**Comment by housseindjirdeh** [comment]
Hmm that's definitely an option, definitely don't want to addd _too_ much stack navigation if its not necessary but I do think it's cool to be able to show all the languages in the repo.

Before anybody embarks on this journey, will love to discuss mocked screenshots of how it might look if possible.

**Comment by andrewda** [comment]
> definitely don't want to add *too* much stack navigation if its not necessary

Yep totally agree. Would love to hear some ideas from others on how this could be improved!

**Comment by FlorianTaut** [comment]
I mean at the end of the grey part. Marked it red in the screen

![Uploading IMG_0371.JPG…]()

**Comment by kevinkong91** [comment]
new to open-source, and would love to take a crack at this one!

**Comment by housseindjirdeh** [comment]
Still think this is something that might be worthy to take a crack at :)

Like @andrewda mentioned: A bar that shows the different repository colours and when tapped can take the user to a separate screen that shows the different language colours is an option. We can make use of a modal screen for iOS here.

Happy to defer how it actually looks design wise to whoever picks up the ticket. But please post screenshots on what you may be envisioning before going full in just so everybody is on board!!

**Comment by daggala** [comment]
But what about displaying the languages in a accordion list, dropdown. So when the bar is being pressed, the list would be displayed and disappear when it's pressed again. Maybe only containing max fx. 8 languges so that it will always fit on the smallest screens.The rest of the languages could be named "Other" and beside we could display the remaining percentage of remaining languages. What do you think?

**Comment by t-laird** [comment]
Hi! I’m a first-time contributor and was hoping to help out with this issue. I noticed that there hasn't been much activity on this issue since October but understand that there may be a solution in progress already and would be happy to help elsewhere if so. Thanks!

**Comment by daggala** [comment]
Hello @t-laird and welcome! I am working currently on this issue, slowly getting there and learning as I move on like a snail. If you sign in to the chat channel you could ask around which tasks might suit a first-time contirbutor :)
